server/src/services/cleanVoteInput.js

```diff
@@ -0,0 +1,19 @@
+const cleanVoteInput = formInput => {
+  Object.keys(formInput).forEach(field => {
+    console.log(field)
+    
+    if(field === "userId") {
+        const validId = Number(formInput.userId)
+        formInput.userId = validId
+    }
+
+    if(field === "reviewId") {
+        const validId = Number(formInput.reviewId)
+        formInput.reviewId = validId
+    }
```

Comment:
Take a look at the comment left in `cleanUserInput` for combining these statements to DRY your code